internal/serialization/serialization.go

```diff
@@ -74,25 +77,24 @@ func (s *Service) ToData(object interface{}) (r *Data, err error) {
 }
 
 // ToObject deserializes the given Data to an object.
-// It can safely be called on an object that is already deserialized. In that case, that instance
-// is returned.
-// If this is called with nil, nil is returned.
+// nil is returned if called with nil.
 func (s *Service) ToObject(data *Data) (r interface{}, err error) {
 	defer func() {
 		if rec := recover(); rec != nil {
 			err = makeError(rec)
 		}
 	}()
+	var ok bool
```

Comment:
Is there a reason we define this at the top, even if we might not need it?

Reply:
Yes, panics can only be caught by deferred functions.

Reply:
Oh, I meant the `var ok bool` definition

Reply:
The compiler would probably optimize that.